package.json

```diff
@@ -45,62 +45,18 @@
     ]
   },
   "devDependencies": {
-    "@babel/core": "^7.6.2",
-    "@babel/runtime": "^7.6.2",
-    "@types/jest": "^25.1.4",
-    "babel-jest": "^26.1.0",
-    "codecov": "^3.6.5",
+    "chalk": "^4.1.0",
+    "node-fetch": "^2.6.0",
     "husky": "^3.0.0",
-    "jest": "^26.1.0",
-    "jest-circus": "^26.1.0",
-    "jest-junit": "^10.0.0",
-    "jest-snapshot": "^26.1.0",
     "lerna": "^3.16.0",
-    "patch-package": "^5.1.1",
     "prettier": "1.19.1",
-    "prettier-plugin-java": "^0.6.0",
-    "prettier-plugin-solidity": "1.0.0-alpha.35",
     "pretty-quick": "^2.0.1",
-    "solc": "0.5.8",
-    "ts-jest": "^26.1.1",
-    "ts-node": "^8.3.0",
-    "tsconfig-paths": "^3.8.0",
-    "tslint": "^5.20.0",
-    "typescript": "^3.8.3",
-    "typescript-tslint-plugin": "^0.5.4"
+    "minimist": "^1.2.5",
+    "ts-node": "^8.10.2",
+    "typescript": "^3.8.3"
   },
   "dependencies": {},
   "resolutions": {
-    "**/truffle/mocha": "^6.2.2",
```

Comment:
These resolutions exist mostly to workaround bugs and/or security vulnerabilities in transitive dependencies (deps of deps). What's the downside in leaving these here?

Reply:
+1 to Rossy's concern about the transitive dependencies. Is the idea to have those resolutions moved to other packages?

Reply:
which bug? since when did it exist? is it fixed? where is it documented?

Reply:
need more information to answer that question

Reply:
what about packages using jest instead of mocha?

Reply:
ah now i understand why it has so many resolutions, they are used to fix vulnerabilities. The problem with this technique is that the yarn.lock will get addicted to resolutions and the real problem (of upgrading the package that actually has the reference) will be shadowed.